[2024-08-04T06:59:08.203+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-04T06:59:08.234+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_process.etl_process.load_r2_data_backup scheduled__2024-08-03T00:00:00+00:00 [queued]>
[2024-08-04T06:59:08.243+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_process.etl_process.load_r2_data_backup scheduled__2024-08-03T00:00:00+00:00 [queued]>
[2024-08-04T06:59:08.244+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-04T06:59:08.257+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): etl_process.load_r2_data_backup> on 2024-08-03 00:00:00+00:00
[2024-08-04T06:59:08.265+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_process', 'etl_process.load_r2_data_backup', 'scheduled__2024-08-03T00:00:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpllrpaole']
[2024-08-04T06:59:08.267+0000] {standard_task_runner.py:91} INFO - Job 7: Subtask etl_process.load_r2_data_backup
[2024-08-04T06:59:08.268+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=148) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-04T06:59:08.269+0000] {standard_task_runner.py:63} INFO - Started process 165 to run task
[2024-08-04T06:59:08.319+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_process.etl_process.load_r2_data_backup scheduled__2024-08-03T00:00:00+00:00 [running]> on host 28819dc9564a
[2024-08-04T06:59:08.405+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ztamnaja' AIRFLOW_CTX_DAG_ID='etl_process' AIRFLOW_CTX_TASK_ID='etl_process.load_r2_data_backup' AIRFLOW_CTX_EXECUTION_DATE='2024-08-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-03T00:00:00+00:00'
[2024-08-04T06:59:08.407+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-04T06:59:08.503+0000] {base.py:84} INFO - Using connection ID 'r2_backup_lake_conn' for task execution.
[2024-08-04T06:59:08.842+0000] {logging_mixin.py:188} INFO - shopsmart products/08-04-24-products.csv
[2024-08-04T06:59:09.392+0000] {logging_mixin.py:188} INFO - Successfully uploaded products/08-04-24-products.csv to R2
[2024-08-04T06:59:09.394+0000] {logging_mixin.py:188} INFO - shopsmart transactions/08-04-24-transactions.csv
[2024-08-04T06:59:09.776+0000] {logging_mixin.py:188} INFO - Successfully uploaded transactions/08-04-24-transactions.csv to R2
[2024-08-04T06:59:09.777+0000] {logging_mixin.py:188} INFO - shopsmart sales/08-04-24-sales.csv
[2024-08-04T06:59:10.217+0000] {logging_mixin.py:188} INFO - Successfully uploaded sales/08-04-24-sales.csv to R2
[2024-08-04T06:59:10.219+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-04T06:59:10.221+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-04T06:59:10.235+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_process, task_id=etl_process.load_r2_data_backup, run_id=scheduled__2024-08-03T00:00:00+00:00, execution_date=20240803T000000, start_date=20240804T065908, end_date=20240804T065910
[2024-08-04T06:59:10.292+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-04T06:59:10.320+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-04T06:59:10.326+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
